cmd/monaco/generate/deletefile/deletefile.go

```diff
@@ -36,7 +36,7 @@ import (
 	"strings"
 )
 
-func createDeleteFile(fs afero.Fs, manifestPath string, projectNames, specificEnvironments []string, filename, outputFolder string) error {
+func createDeleteFile(fs afero.Fs, manifestPath string, projectNames, specificEnvironments []string, filename string, includeTypes []string, excludeTypes []string, outputFolder string) error {
```

Comment:
Should we create a struct here to pass the data through? We wire it through quite some methods.